_posts/en/newsletters/2025-11-07-newsletter.md

```diff
@@ -0,0 +1,93 @@
+---
+title: "Bitcoin Optech Newsletter #379"
+permalink: /en/newsletters/2025/11/07/
+name: 2025-11-07-newsletter
+slug: 2025-11-07-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+
+FIXME:bitschmidty
+
+## News
+
+FIXME:bitschmidty
+
+- **Comparing performance of ECDSA signature validation in OpenSSL vs. libsecp256k1** Sebastian
+  Falbesoner [posted to Delving][sebastion delving] about comparing the performance of ECDSA
+  signature validation between OpenSSL and libsecp256k1 over the last decade. He mentions that it
+  will be the 10th anniversary of Bitcoin Core switching from OpenSSL to libsecp256k1, and wanted to
+  imagine a hypothetical situation in which Bitcoin Core never switched.
+
+  From the start, when the switch happened, libsecp256k1 was 2.5 - 5.5 times faster than OpenSSL.
+  Nonetheless, OpenSSL could have improved over the years, so it was worth testing to see how it
+  compared over the decade.
+
+  The methodology was tested using 3 states that can be tested with functions in both libraries. A
+  benchmark was made with a list of pseudo-random key pairs. He ran the benchmark on 3 separate
+  machines and provided a bar plot. The chart showed that over the years, libsecp256k1 had improved
+  significantly while OpenSSL had remained the same.
+
+  Sebastion makes the conclusion that outside the Bitcoin ecosystem, the secp256k1 curve is not that
+  relevant and doesn't count as a first-class citizen, which justifies hours of work to improve. He
+  also encourages readers to try to reproduce his results and to report any issues in his
+  methodology or differences found in their results. The [source
+  code][libsecp benchmark code] is available on GitHub.
+
+## Changing consensus
+
+_A monthly section summarizing proposals and discussion about changing
+Bitcoin's consensus rules._
+
+FIXME:bitschmidty
+
+- **BIP54 implementation and test vectors**: Antoine Poinsot [posted][ap bip54
+  post] on the Bitcoin-Dev mailing list an update on his [Consensus
+  Cleanup][topic consensus cleanup] work, now [BIP54][] (see [Newsletter
+  #348][news348 bip54] for more details).
+
+  [PR99][binq bip54 pr] was opened against [Bitcoin Inquisition][bitcoin
+  inquisition repo] v29.1, implementing BIP54 consensus rules. This PR comes
```

Comment:
is it open to Bitcoin Inquisition or specifically against the 29.1 version? Unclear what it means to be against 29.1 here